moped-editor/package.json

```diff
@@ -83,10 +81,8 @@
     "lodash.toarray": "^4.4.0",
     "mapbox-gl": "^3.0.1",
     "papaparse": "^5.4.1",
-    "patch-package": "^8.0.1",
-    "prop-types": "^15.8.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
i read aobut the [automatic state update batching](https://react.dev/blog/2022/03/08/react-18-upgrade-guide#automatic-batching) in React v18 and it was a TIL. I took a brief look at places where we batch update like in `ProjectNotes.js`, and everything is working as expected on the deploy preview. Nothing but performance improvements from here! 🙌 🙌 🙌